src/storage/filesystem.py

```diff
@@ -1,94 +1,128 @@
+"""
+src/storage/filesystem.py
+=========================
+"""
 from binascii import hexlify, unhexlify
 from os import listdir, makedirs, path, remove, rmdir
 import string
 from threading import RLock
 import time
-import traceback
 
 from paths import lookupAppdataFolder
 from storage import InventoryStorage, InventoryItem
 
-class FilesystemInventory(InventoryStorage):
+
+class FilesystemInventory(InventoryStorage):    # pylint: disable=too-many-ancestors, abstract-method
+    """Module for using filesystem (directory with files) for inventory storage"""
     topDir = "inventory"
     objectDir = "objects"
     metadataFilename = "metadata"
     dataFilename = "data"
 
     def __init__(self):
-        super(self.__class__, self).__init__()
+        super(FilesystemInventory, self).__init__()
         self.baseDir = path.join(lookupAppdataFolder(), FilesystemInventory.topDir)
         for createDir in [self.baseDir, path.join(self.baseDir, "objects")]:
             if path.exists(createDir):
                 if not path.isdir(createDir):
                     raise IOError("%s exists but it's not a directory" % (createDir))
             else:
                 makedirs(createDir)
-        self.lock = RLock() # Guarantees that two receiveDataThreads don't receive and process the same message concurrently (probably sent by a malicious individual)
+        self.lock = RLock()
+        # Guarantees that two receiveDataThreads don't receive and process the same message
```

Comment:
This refers to `self.lock` and according to PEP8 block comments should be **above** the code they refer to.

Reply:
Okay, will amend the changes. Thank you.